src/components/Maps/InfoWindow.tsx

```diff
@@ -47,33 +54,38 @@ const InfoWindow = ({
       onClose={handleClose}
       BackdropProps={{ style: { backgroundColor: 'transparent' } }}
     >
-      <DialogContent style={{ overflow: 'hidden' }}>
-        <Typography variant="body1" color="textSecondary">
-          Name:
-          <span
-            style={{ display: 'inline', color: '#000000' }}
-            color="textPrimary"
-          >
-            {` ${title} `}
-          </span>
-        </Typography>
-        <Typography variant="body1" color="textSecondary">
-          Location:
-          <span
-            style={{ display: 'inline', color: '#000000' }}
-            color="textPrimary"
-          >
-            {` ${address} `}
-          </span>
-        </Typography>
-      </DialogContent>
-      <DialogActions>
-        <Button size="small" classes={{ root: infoStyles.dialogDelete }}>
-          Delete
-        </Button>
-        <Button size="small" classes={{ root: infoStyles.dialogButton }}>
-          Edit
-        </Button>
+      <Container classes={{ root: infoStyles.flexContainer }}>
+        <DialogContent style={{ overflow: 'hidden', paddingLeft: '0px' }}>
+            <Typography variant="body1" color="textSecondary">
+              Name:
+              <span
+                style={{ display: 'inline', color: Colours.Black }}
+                color="textPrimary"
+              >
+                {` ${title} `}
+              </span>
+            </Typography>
+            <Typography variant="body1" color="textSecondary">
+              Location:
+              <span
+                style={{ display: 'inline', color: Colours.Black }}
+                color="textPrimary"
+              >
+                {` ${address} `}
+              </span>
+            </Typography>
+        </DialogContent>
+        <CloseIcon onClick={handleClose} classes={{ root: infoStyles.closeButton }}/>
+      </Container>
+      <DialogActions style = {{ padding: '0px'}}>
+        <Container classes={{ root: infoStyles.flexContainer }}>
```

Comment:
Can you add some padding for extra space below the buttons? (see Figma)

![image](https://user-images.githubusercontent.com/22457859/97376539-01d69a80-1894-11eb-8fc8-9c6f371f2a27.png)